Midi.ahk

```diff
@@ -258,7 +407,21 @@ Class Midi
         __OpenMidiIn( midiInDeviceId )        
       }
 
```

Comment:
It seems to need `Return` between `__SelectMidiInDevice` end and before `__SelectMidiOutDevice`
```suggestion
Return

```